server/translations.go

```diff
@@ -0,0 +1,27 @@
+package server
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mattermost/mattermost-mattermod/model"
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+func (s *Server) handleTranslationPr(pr *model.PullRequest) {
+	if pr.Username != s.Config.Username {
+		return
+	}
+
+	dataMsg := fmt.Sprintf("####[%v translations PR %v](%v)\n", pr.RepoName, time.Now().UTC().Format(time.RFC3339), pr.URL)
+	msg := dataMsg + s.Config.TranslationsMattermostMessage
+	mlog.Debug("Sending Mattermost message", mlog.String("message", msg))
+
+	webhookRequest := &Payload{Username: "Weblate", Text: msg}
+	r, err := s.sendToWebhook(s.Config.TranslationsMattermostWebhookURL, webhookRequest)
+	if err != nil {
+		mlog.Error("Unable to post to Mattermost webhook", mlog.Err(err))
+		return
+	}
+	defer r.Body.Close()
```

Comment:
Any reason to use `defer` as the last statement of a function? Or do we expect more lines to be added later?

Also, please nil-check `Body` before calling `Close`.

Lastly, let's read the body to completion to reuse the same connection.